app/models/user.rb

```diff
@@ -113,9 +113,11 @@ class User < ApplicationRecord
            through: :followings,
            source: :user
 
+  class_attribute :update_wrangling_activity
```

Comment:
Personally, I'd prefer `thread_cattr_accessor` here. It's not relevant for production, because each Unicorn worker is a separate process, but I believe this could cause issues if someone wanted to run a multi-threaded server instead.